source/agora/node/Ledger.d

```diff
@@ -116,7 +118,7 @@ public class Ledger
                 Block block;
                 this.storage.readBlock(block, this.last_block.header.height - i);
                 this.enroll_man.restoreValidators(this.last_block.header.height,
-                    block, this.utxo_set.getUTXOFinder());
+                    block, this.utxo_set.getUTXOFinder(), utxos);
```

Comment:
Is this just restoring the current node's enrollments? What about other nodes' enrollments?

Reply:
They are restored already (see L551).

Reply:
L551? In this PR? It's pointing to `auto pool = new TransactionPool(":memory:");`

Reply:
He means L551 of the EnrollmentManager.d

Reply:
Right, that looks ok then.